### ISSUE TITLE
Better text wrapping on select moods page for mobile site

### ISSUE THREAD

**Issue by IamSuperB4** [info442-aut21/d-team#94]
Again, I know this is similar to #92 , but this page was styled differently, and we have longer words like "entertainment" to consider how to make that text look proper. This issue will be tougher than #92 , and would probably need to be done after #93 , which is another tough task.

**Comment by IamSuperB4** [comment]
Note to @palashgujrathi , even though this page looks the worst on mobile, I would recommend trying to work on this issue last and making it a stretch goal, as well as #93 , since me and Judy worked on this for the desktop site and it was a huge pain. The mobile site won't be any easier
